opennms-webapp/src/main/webapp/lib/bootstrap-sass-official/assets/stylesheets/bootstrap/_variables.scss

```diff
@@ -355,7 +355,7 @@ $container-lg:                 $container-large-desktop !default;
 //##
 
 // Basics of a navbar
-$navbar-height:                    50px !default;
+$navbar-height:                    65px !default;
```

Comment:
If we need to update this variable, we should set it on our theme instead of modifying it here.